features/promotion/receiving_discount_based_on_cart_quantity.feature

```diff
@@ -5,36 +5,36 @@ Feature: Receiving discount based on cart quantity
     I want to have promotion applied to my cart when my cart quantity qualifies
 
     Background:
-        Given the store is operating on a single "France" channel
+        Given the store operates on a single channel in "France"
         And the store has a product "PHP T-Shirt" priced at "€100.00"
         And there is a promotion "Holiday promotion"
 
-    @todo
+    @ui
     Scenario: Receiving discount when buying more than required quantity
-        Given the promotion gives "€10.00" fixed discount to every cart with quantity at least 3
+        Given the promotion gives "€10.00" fixed discount to every order with quantity at least 3
```

Comment:
I've tried to be consistent with previously implemented steps. There is already step "And it gives "€10.00" fixed discount **to every order**". Do you think ``cart`` is more correct and it should be changed in previous steps?